graylog2-server/src/main/java/org/graylog2/alarmcallbacks/AlarmCallbackConfigurationServiceImpl.java

```diff
@@ -37,14 +37,14 @@
 import java.util.Map;
 
 public class AlarmCallbackConfigurationServiceImpl implements AlarmCallbackConfigurationService {
-    private final JacksonDBCollection<AlarmCallbackConfigurationImpl, String> coll;
+    private final JacksonDBCollection<AlarmCallbackConfigurationImpl, ObjectId> coll;
```

Comment:
Are these `String` vs `ObjectId` changes related to the MongoDB driver update or what is the intention behind those?

Reply:
Without these changes, MongoJack fails with a `ClassCastException` in the [`WriteResult#getSavedId()`](http://mongojack.org/apidocs/org/mongojack/WriteResult.html#getSavedId--) method.

Similar to https://github.com/mongojack/mongojack/issues/8

Reply:
Scratch that. The problem seems to be Jackson 2.9.x.

I've reverted the commit updating Jackson and added back the commit updating the MongoDB Java driver.